src/react-styleguide/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "react-styleguide",
+  "version": "0.1.0",
+  "private": true,
+  "devDependencies": {
+    "react-scripts": "0.7.0"
+  },
+  "dependencies": {
+    "font-awesome": "^4.7.0",
+    "react": "^15.4.1",
+    "react-dom": "^15.4.1",
+    "tachyons": "^4.5.6",
+    "tachyons-background-position": "^1.0.3",
+    "tachyons-background-size": "^5.0.4",
+    "tachyons-base": "^1.2.6",
+    "tachyons-border-colors": "^4.2.3",
+    "tachyons-border-radius": "^5.1.2",
+    "tachyons-border-style": "^4.0.3",
+    "tachyons-border-widths": "^3.0.3",
+    "tachyons-borders": "^3.0.3",
+    "tachyons-box-shadow": "^2.0.3",
+    "tachyons-box-sizing": "^3.1.9",
+    "tachyons-clears": "^3.0.3",
+    "tachyons-colors": "^5.3.2",
+    "tachyons-coordinates": "^4.0.3",
+    "tachyons-display": "^5.0.3",
+    "tachyons-flexbox": "^2.0.3",
+    "tachyons-floats": "^3.0.3",
+    "tachyons-font-family": "^4.3.2",
+    "tachyons-font-style": "^4.0.3",
+    "tachyons-font-weight": "^5.0.3",
+    "tachyons-forms": "^3.1.2",
+    "tachyons-heights": "^6.1.2",
+    "tachyons-hovers": "^2.4.5",
+    "tachyons-images": "^1.0.10",
+    "tachyons-letter-spacing": "^3.0.3",
+    "tachyons-line-height": "^3.0.5",
+    "tachyons-links": "^3.0.8",
+    "tachyons-lists": "^2.0.11",
+    "tachyons-max-widths": "^4.0.3",
+    "tachyons-opacity": "^1.1.7",
+    "tachyons-outlines": "^1.0.2",
+    "tachyons-overflow": "^4.0.3",
+    "tachyons-position": "^6.0.3",
+    "tachyons-skins": "^4.0.2",
+    "tachyons-skins-psuedo": "^1.0.0",
+    "tachyons-spacing": "^6.0.3",
+    "tachyons-tables": "^1.0.6",
+    "tachyons-text-align": "^3.0.3",
+    "tachyons-text-decoration": "^4.0.3",
+    "tachyons-text-transform": "^4.0.3",
+    "tachyons-type-scale": "^6.0.3",
+    "tachyons-typography": "^3.0.3",
+    "tachyons-utilities": "^1.2.8",
+    "tachyons-vertical-align": "^4.0.3",
+    "tachyons-visibility": "^2.0.3",
+    "tachyons-white-space": "^4.0.4",
+    "tachyons-widths": "^5.0.3",
+    "tachyons-word-break": "^3.0.3",
+    "tachyons-z-index": "^1.0.6"
```

Comment:
@trevordmiller I honestly don't know :(  I was going for parity with the packages that @DrShpongle had in the source material... 

@DrShpongle can you please weigh in?